notebook/zeroshot_lightgbm.ipynb

```diff
@@ -0,0 +1,553 @@
+    "When `estimator_list` is omitted, the most promising estimator together with its hyperparameter configuration will be tried first, which are both decided by zero-shot automl."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[flaml.automl: 05-31 22:54:25] {2373} INFO - task = regression\n",
+      "[flaml.automl: 05-31 22:54:25] {2375} INFO - Data split method: uniform\n",
+      "[flaml.automl: 05-31 22:54:25] {2379} INFO - Evaluation method: cv\n",
+      "[flaml.automl: 05-31 22:54:25] {2448} INFO - Minimizing error metric: 1-r2\n",
+      "[flaml.automl: 05-31 22:54:25] {2586} INFO - List of ML learners in AutoML Run: ['lgbm']\n",
+      "[flaml.automl: 05-31 22:54:25] {2878} INFO - iteration 0, current learner lgbm\n",
+      "[flaml.automl: 05-31 22:56:54] {3008} INFO - Estimated sufficient time budget=1490299s. Estimated necessary time budget=1490s.\n",
+      "[flaml.automl: 05-31 22:56:54] {3055} INFO -  at 149.1s,\testimator lgbm's best error=0.1513,\tbest estimator lgbm's best error=0.1513\n",
+      "[flaml.automl: 05-31 22:56:54] {2878} INFO - iteration 1, current learner lgbm\n",
+      "[flaml.automl: 05-31 22:59:24] {3055} INFO -  at 299.0s,\testimator lgbm's best error=0.1513,\tbest estimator lgbm's best error=0.1513\n",
+      "[flaml.automl: 05-31 22:59:24] {2878} INFO - iteration 2, current learner lgbm\n",
+      "[flaml.automl: 05-31 23:01:34] {3055} INFO -  at 429.1s,\testimator lgbm's best error=0.1513,\tbest estimator lgbm's best error=0.1513\n",
+      "[flaml.automl: 05-31 23:01:34] {2878} INFO - iteration 3, current learner lgbm\n",
+      "[flaml.automl: 05-31 23:04:43] {3055} INFO -  at 618.2s,\testimator lgbm's best error=0.1513,\tbest estimator lgbm's best error=0.1513\n",
+      "[flaml.automl: 05-31 23:05:14] {3315} INFO - retrain lgbm for 31.0s\n",
+      "[flaml.automl: 05-31 23:05:14] {3322} INFO - retrained model: LGBMRegressor(colsample_bytree=0.7019911744574896,\n",
+      "              learning_rate=0.02263575841107852, max_bin=511,\n",
+      "              min_child_samples=2, n_estimators=4797, num_leaves=122,\n",
+      "              reg_alpha=0.004252223402511765, reg_lambda=0.11288241427227633,\n",
+      "              verbose=-1)\n",
+      "[flaml.automl: 05-31 23:05:14] {2617} INFO - fit succeeded\n",
+      "[flaml.automl: 05-31 23:05:14] {2618} INFO - Time taken to find the best model: 149.06516432762146\n"
+     ]
+    }
+   ],
+   "source": [
+    "from flaml import AutoML\n",
+    "\n",
+    "automl = AutoML()\n",
+    "settings = {\n",
+    "    \"task\": \"regression\",\n",
+    "    \"starting_points\": \"data\",\n",
+    "    \"estimator_list\": [\"lgbm\"],\n",
+    "    \"time_budget\": 600,\n",
+    "}\n",
+    "automl.fit(X_train, y_train, **settings)"
```

Comment:
Do we need to mention somewhere, that when `time_budget` and `max_iter` are unspecified, AutoML is effectively zero-short AutoML?

Reply:
See the text above.